modules/editor-one/assets/scss/_common.scss

```diff
@@ -0,0 +1,266 @@
+@import "../../../../assets/dev/scss/helpers/variables";
+@import "../../../../assets/dev/scss/helpers/mixins";
+@import "variables";
+@import "../../../../assets/dev/scss/direction/theme-light";
+
+@import "theme-light";
+
+@media (prefers-color-scheme: dark) {
+
+}
+
+#wpcontent,
+#wpbody-content,
+#wpbody-content > * {
+	background-color: var(--e-one-palette-background-default) !important;
+}
+
+body.wp-admin {
+	background-color: var(--e-one-palette-background-default) !important;
+	font-family: var(--e-one-typography-fontFamily);
+}
+
+.wrap {
+	background-color: var(--e-one-palette-background-default) !important;
+	margin: 20px;
+	max-width: 100%;
+}
+
+#wpbody-content .wrap,
+.elementor-settings-form-page,
+.elementor-settings-form-page > *,
+.form-table,
+.form-table th,
+.form-table td,
+.form-table tr {
+	background-color: var(--e-one-palette-background-default) !important;
+}
+
+.postbox,
+.metabox-holder,
+.postbox .inside {
+	background-color: var(--e-one-palette-background-default) !important;
+}
+
+.wrap {
+	h1.wp-heading-inline {
+		font-family: var(--e-one-typography-fontFamily);
+		font-weight: 700;
+		font-size: 24px;
+		color: var(--e-one-palette-text-primary);
+		margin: 0;
+		margin-inline-end: 20px;
+		padding: 0;
+		line-height: 1.2;
+	}
+}
+
+.e-admin-header {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin-bottom: 20px;
+	padding-bottom: 20px;
+	border-bottom: 1px solid var(--e-one-palette-divider);
+	border-bottom-color: var(--e-one-palette-divider);
+
+	.e-admin-actions {
+		display: flex;
+		gap: 10px;
+		align-items: center;
+	}
+}
+
+.button-primary,
```

Comment:
Generally speaking we try to avoid the use of `!important`, but I can imagine that we need it, because other plugins might be using custom styling as well.

Can you elaborate about this, Maks?

My first question would be, have you tried to increase the specificity e.g. by adding:
`body.wp-admin .button-primary{}`?